File: Practice/pi.jl
# Calculate π using Taylor Series

o = time()
s = 0.0

for n in 0:10000000
#    println(((-1)^n//(2n+1)))
    global s = s + (-1)^n/(2n+1)
end

pi = 4s
π = 4s
t = time()

#println(pi)
println(π)
println("Time Interval: ",t-o," second")

File: Series Practice/pi.py
# Calculate π using Taylor Series

import time

o = time.time()
s = 0.0
n = 0

for n in range(10000001):
    s = s + (-1)**n/(2*n+1)

pi = 4*s
t = time.time()

print(pi)
print("Time Interval:",t-o,"second")